src/gui/Src/Gui/DataCopyDialog.cpp

```diff
@@ -3,6 +3,7 @@
 #include "Bridge.h"
 #include <QCryptographicHash>
 #include <QTextCodec>
+#include <QMessageBox>

```

Comment:
Where is this message box used?

Reply:
debugging purposes...I forgot to take it out xD